lib/ast_grammar.re

```diff
@@ -1,26 +1,34 @@
 /* Essence of grammar.json content */
 
+[@deriving show]
```

Comment:
what does @deriving show do?

Reply:
It auto generates the Ast_grammar.show_xxx function, a bit like what console.log(anything) can do in Javascript. I use this show_xxx function in test_tree_sitter.re